src/main/target/GEPRC_F722_AIO/target.h

```diff
@@ -53,6 +57,8 @@
 
 // *************** I2C/Baro/Mag *********************
 #define USE_I2C
```

Comment:
USE_I2C should probably be inside the ifdef bellow as well

Reply:
> USE_I2C should probably be inside the ifdef bellow as well

Done.